modules/_labs/select/react/lib/SelectBase.tsx

```diff
@@ -352,6 +368,11 @@ export default class SelectBase extends React.Component<SelectBaseProps> {
         value: option.value,
         ...(onOptionSelection
           ? {
+              // Prevent click from propagating (and, say, dismissing
+              // a modal containing the Select)
+              onClick: (event: React.MouseEvent) => {
+                event.stopPropagation();
+              },
```

Comment:
We will need a better solution here. We have the same issue with other Popup systems and we shouldn't have to solve it one-off in all of them. We should add some requirements and fix it here: #182

Reply:
Removed stopPropagation call after merging in the stack manager.